src/games/strategy/engine/framework/mapDownload/DownloadRunnable.java

```diff
@@ -52,26 +59,45 @@ public void setError(final String error) {
 
   @Override
   public void run() {
```

Comment:
Local file is useful for development, the other two locations require correctness and check-ins.